packages/fluentui/docs/src/index.ejs

```diff
@@ -24,11 +23,6 @@
       }
     })()
   </script>
-  <style>
-    html {
-      font-size: 14px;
```

Comment:
nit: as Teams theme is 14px-based, having `rem` 14px-based makes debugging easier.

Reply:
reverted this as it causes screener regressions because of rounding